src/msg/async/compression_onwire.h

```diff
@@ -97,7 +97,7 @@ namespace ceph::compression::onwire {
     static rxtx_t create_handler_pair(
       CephContext* ctx,
       const CompConnectionMeta& comp_meta,
-      std::uint64_t compress_min_size);
```

Comment:
I wasn't able to understand why std:: is a problem with this gcc bug. If it is not - I'd rather not remove the std's

Reply:
I took that change from the patch in the description of https://tracker.ceph.com/issues/58477 so I'm not sure of the rationale. I've just tried without removing `std::` and the build failed because `error: 'std::uint64_t' has not been declared`. I reckon I can fix that with another `#include <cstdint>`...

Reply:
Yup, turns out we don't need to drop the `std::` bits.